packages/backend/src/apiV1.ts

```diff
@@ -104,6 +114,49 @@ apiV1Router.get('/user', isAuthenticated, async (req, res) => {
     });
 });
 
+apiV1Router.post('/user', isAuthenticated, isNotDemo, async (req, res, next) =>
+    UserModel.updateProfile(req.user!.userId, req.user!.email, req.body)
+        .then((user) => {
+            res.json({
+                status: 'ok',
+                results: user,
+            });
+        })
+        .catch(next),
+);
+
+apiV1Router.post(
+    '/user/password',
+    isAuthenticated,
+    isNotDemo,
+    async (req, res, next) =>
+        UserModel.updatePassword(req.user!.userId, req.user!.userUuid, req.body)
+            .then(() => {
+                req.logout();
+                req.session.save((err) => {
+                    if (err) {
+                        next(err);
+                    } else {
+                        res.json({
+                            status: 'ok',
+                        });
+                    }
+                });
+            })
+            .catch(next),
+);
+
+apiV1Router.post('/org', isAuthenticated, isNotDemo, async (req, res, next) =>
```

Comment:
`/user/me` makes sense but I'm not sure about `/org/me` I thin kwe should keep it `/org` for now since we only support 1 org and later make it `/org/{uuid}`

Reply:
perfect!